### DISCUSSION TITLE
Support for LCDs using SSD1803 controller IC?

### DISCUSSION THREAD

**Discussion by ddevries6161** [forntoh/LcdMenu#146] [Q&A] [answered]
Hello,
I'm using Electronic Assemblies DOGM204 20x4 LCD (SSD1803 controller IC) with a Nano ESP32 board.  SSD1803A_I2C library works to control the display but I don't think it is compatible with LiquidCrystal_I2C.  Would you consider adding support for this family of displays to LcdMenu?  If you have another suggestion for how I can use LcdMenu with this LCD, I'd appreciate any advice. 

Thank you!

**Comment by forntoh** [comment, marked as answer]
@ddevries6161 Added #243
